src/app/src/main/java/com/example/manas/alias/MainActivity.java

```diff
@@ -0,0 +1,13 @@
+package com.example.manas.alias;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+    }
+}
```

Comment:
Wrong file. java/com/example/manas shouldn;'t exist. Edit the main file instead.